scene/gui/nine_patch_rect.cpp

```diff
@@ -89,11 +90,27 @@ void NinePatchRect::_bind_methods() {
 	BIND_ENUM_CONSTANT(AXIS_STRETCH_MODE_TILE_FIT);
 }
 
+void NinePatchRect::_texture_changed() {
+	queue_redraw();
+	update_minimum_size();
+	emit_signal(SceneStringNames::get_singleton()->texture_changed);
```

Comment:
Wasn't sure if this should be called each time, but the documentation says "when the texture changes" so included it

Reply:
I think it's more expected that the signal gets emitted only when the texture property changes to a different texture. The description just isn't very precise.

Although I wonder why this signal even exists. Other control nodes don't have anything like that.

Reply:
Will update tomorrow if desired

Reply:
Done!